sdk/src/androidTest/java/com/mapbox/maps/CameraAnimationsPluginTest.kt

```diff
@@ -901,6 +954,6 @@ class CameraAnimationsPluginTest : BaseAnimationMapTest() {
   companion object {
     private const val TAG = "Mbgl-CameraManager"
     private const val EPS = 0.000001
-    private const val LATCH_MAX_TIME = 5_000L
+    private const val LATCH_MAX_TIME = 6_000L
```

Comment:
Why 6?)

Reply:
sorry, test for not getting the test to pass, is now removed